idd/Energy+.idd.in

```diff
@@ -24351,6 +24356,7 @@ Exterior:FuelEquipment,
        \required-field
        \type choice
        \key Electricity
+       \key Water
```

Comment:
It appears in the code to work, at least it validates it and sets up output variables... https://github.com/NREL/EnergyPlus/blob/db4decb82c9c3c9310938c202a2f62a97e75c5d6/src/EnergyPlus/ExteriorEnergyUse.cc#L372

But we can keep it out if that makes more sense. In general, most of these changes are to bring the IDD and code closer together.